dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -3693,17 +3714,7 @@ def test_null_calculation_with_differently_sampled_cols(self):
         self.assertEqual(0.5, profile2._get_row_is_null_ratio())
         self.assertEqual(1, profile2._get_row_has_null_ratio())
 
-    @mock.patch(
-        "dataprofiler.profilers.profile_builder." "ColumnPrimitiveTypeProfileCompiler"
-    )
-    @mock.patch("dataprofiler.profilers.profile_builder." "ColumnStatsProfileCompiler")
-    @mock.patch("dataprofiler.profilers.profile_builder." "ColumnDataLabelerCompiler")
-    @mock.patch("dataprofiler.profilers.profile_builder.DataLabeler")
-    @mock.patch(
-        "dataprofiler.profilers.profile_builder."
-        "StructuredProfiler._update_correlation"
-    )
-    def test_null_row_stats_correct_after_updates(self, *mocks):
+    def test_null_row_stats_correct_after_updates(self):
         data1 = pd.DataFrame([[1, None], [1, 1], [None, None], [None, 1]])
```

Comment:
Won't removing this cause this test to run all the options as opposed ot just testing what is desired since this is using default `ProfilerOptions`?